### ISSUE TITLE
2.0.9-pre6 is not able to decompile METADATA\REALITY\TABLES\NMS_REALITY_GCTECHNOLOGYTABLE.MBIN

### ISSUE THREAD

**Issue by clampi** [monkeyman192/MBINCompiler#139]
**Description**
Current version is not able to decompile "METADATA\REALITY\TABLES\NMS_REALITY_GCTECHNOLOGYTABLE.MBIN"
Here is the output of the operation attempt
[WARN]:` File not recognized. You may need to use an older version of MBINCompiler.

  [INFO]: d:\Games\Editors\NMS_auto_modbuilder\MODBUILDER\Extras\ExtractAndDecompileMod\METADATA\REALITY\TABLES\NMS_REALITY_GCTECHNOLOGYTABLE.MBIN

[ERROR]: [AggregateException]: One or more errors occurred.

**Expected Behavior**
Expected it will decompile without an error.

**Steps To Reproduce**
1. Unpack NMSARC.515F1D3.pak
2. Try to decompile NMS_REALITY_GCTECHNOLOGYTABLE.MBIN
3. Encounter the above shown error message.

**Specs (please complete the following information, if applicable):**
 - MBINCompiler v2.0.9.6
 - NMS Game Version or Steam Build Number: The executable has no meaningful version information, don't know how to find.
 - MBIN File: NMS_REALITY_GCTECHNOLOGYTABLE.MBIN, NMS_U3REALITY_GCTECHNOLOGYTABLE.MBIN
 - MBIN File Version: Not sure how to find that

**Additional Information**
File could be successfully decompiled with a previous version that is no longer available for download, 2.10.0.1. There are other MBINs in NMSARC.515F1D3.pak that can't be decompiled, the above are just an excerpt.
[MBINCompiler.log](https://github.com/monkeyman192/MBINCompiler/files/3568249/MBINCompiler.log)

**Comment by Entoarox** [comment]
The 2.10.0.1 was a mistaken assumption, it has since been *renamed* in the downloads list, and can be found as pre5

Next, if you read the description of pre6, you'd notice that it was designed for the currently experimental update (Which has yet to be released on stable)

TL;DR You're using a TBINCompiler version that is newer then your NMS version, use the earlier release.

**Comment by clampi** [comment]
Problem was fixed once game got patched to 2.11. Brought game files to match the version expect by MBINCompiler.
